backend/llm/brainpicking.py

```diff
@@ -59,21 +65,22 @@ class BrainPicking(BaseModel):
     Main class for the Brain Picking functionality.
     It allows to initialize a Chat model, generate questions and retrieve answers using ConversationalRetrievalChain.
     """
-    
+
     # Default class attributes
     llm_name: str = "gpt-3.5-turbo"
     settings = BrainSettings()
+    llm_config = LLMSettings()
```

Comment:
config or settings ?

Reply:
Hmm they are the same thing in my mind. Config is normally an object. Settings are the individual attributes.